tests/data/spe1_st/tests/test_webviz_ert.py

```diff
@@ -6,23 +6,12 @@
     ResponseCorrelation,
     ObservationAnalyzer,
 )
-from tests.conftest import select_first
+from tests.conftest import select_first, setup_plugin
 
 parameter_keys = ["FIELD_PROPERTIES:POROSITY", "FIELD_PROPERTIES:X_MID_PERMEABILITY"]
 response_keys = ["WGPT:PROD", "WWPT:PROD", "WOPT:PROD", "WWIT:INJ"]
 
 
-def _gen_plugin(dash_duo_handle, PluginClass):
-    app = dash.Dash(__name__)
-    plugin = PluginClass(app, project_identifier=None)
-    layout = plugin.layout
-    app.layout = layout
-    dash_duo_handle.start_server(app)
-    windowsize = (630, 2000)
-    dash_duo_handle.driver.set_window_size(*windowsize)
-    return plugin
-
-
```

Comment:
The PR is marking as changes when you delete a function from one file and adds it to another one, e.g. `setup_plugin` and `_gen_plugin` (they are basically the same, just change the function name and file). This may add noise to the PR, as it seems large than it actually is. Do you know a way to make this kind of change cleaner? We can also discuss within the team to check the opportunities there!

Reply:
if you look at the two commits there is one for the refactoring and one for the actual change. I think that makes a nice separation at least

Reply:
yes, you're right of course, @hnformentin - i was perhaps a bit too eager in this PR, partly because I didn't take the time to read all of the surrounding code. we decided earlier that splitting it into two commits and limiting the refactoring should be acceptable.
i promise to refrain from refactoring in issue PRs in the future : )